packages/@aws-cdk/aws-apigateway/lib/api-key.ts

```diff
@@ -81,6 +81,18 @@ export interface ApiKeyProps extends ApiKeyOptions {
  * for Method resources that require an Api Key.
  */
 export class ApiKey extends Resource implements IApiKey {
+
```

Comment:
Can you also remove this exclusion here - https://github.com/aws/aws-cdk/blob/b1b9d23f442b674df97c1aadb4f84cc0251bb826/packages/%40aws-cdk/aws-apigateway/package.json#L118

This exclusion was added because this import method did not exist. Now that you've added this, it can be dropped 🙌